test/base/test_history.py

```diff
@@ -80,45 +85,45 @@ def check_history(self):
                     self.check_load_from_file(start, str(istart))
                     self.check_history_consistency(start)
 
-                # check that we can also aggregrate from multiple files.
+                # check that we can also aggregate from multiple files.
                 # load more results than necessary to check whether this
                 # also works in case of incomplete results.
                 if storage_type is not None:
-                    read_results_from_file(
+                    optimize.read_results_from_file(
                         self.problem, self.history_options,
                         n_starts=n_starts,
                     )
                 else:
                     with pytest.raises(ValueError):
-                        read_results_from_file(
+                        optimize.read_results_from_file(
                             self.problem, self.history_options,
                             n_starts=n_starts,
                         )
 
-    def check_load_from_file(self, start: OptimizerResult, id: str):
-        """Verify we can reconstitute OptimizerResult from csv file"""
-
+    def check_load_from_file(self, start: optimize.OptimizerResult, id: str):
+        """Verify we can reconstitute OptimizerResult from history file"""
+        # TODO other implementations
```

Comment:
which are missing here?

Reply:
I guess just memory now

Reply:
but memory, by definition, is not saved to a file. I think we will have a hard time ever implementing this ;)